rebar.config

```diff
@@ -32,6 +32,6 @@
 
 
 {profiles, [
-            {test, [{deps, [{meck, "0.8.13"}]}]}
+            {test, [{deps, [{meck, "0.8.13"}]}, {erl_opts, [warnings_as_errors]}]}
```

Comment:
What do you think about adding `export_all, nowarn_export_all` here? that would seem convenient and still trip up on any other warnings as we want.

I am good either way, just wondering what you think on this.

Reply:
I prefer to not allow `export_all` in my projects, but it's your call. That's the default, any way, in which sense we wouldn't have to update anything (if that's what you decide, I mean).